### DISCUSSION TITLE
Pagination

### DISCUSSION THREAD

**Discussion by komir2004** [akeeba/panopticon#504] [General]
### Due diligence

- [X] I have searched all open and closed issues for this problem.
- [X] I have searched all open and closed discussions for this problem.
- [X] I have looked at the CHANGELOG to make sure this problem has not already been addressed.
- [X] I agree to the Code of Conduct of this repository

### Describe the issue

I hope you all had a happy New Year's Eve
I have a question for you regarding the default pagination settings in Panopticon.
Is there a way to increase the number of sites displayed per page from 50 to 100? 
I would appreciate it if you could help me with this.

### Panopticon version

1.0.7

### PHP version

8.1.26

### Database type

MySQL

### Database version

8

### Browser

Edge

**Comment by nikosdion** [comment]
Right now, the default limit for all lists is hardcoded.

With version 1.1.0 you will have a new layout for that page, Dashboard, which displays _all_ sites at once. I believe that this will be a much better option for you than increasing the default limit of sites displayed at once in the default tabular view.
